image/templates/helm/shared/templates/_scanner-v4_init.tpl.htpl

```diff
@@ -176,14 +141,11 @@
   {{ include "srox.safeLookup" (list $ $centralDeployment "apps/v1" "Deployment" $.Release.Namespace "central") }}
   {{ if $centralDeployment.result }}
     {{ include "srox.note" (list $ "Detected central running in the same namespace. Not deploying scanner-v4-indexer from this chart and configuring sensor to use existing scanner-v4-indexer instance, if any.") }}
-    {{ $_ := set $._rox.sensor.localImageScanning "enabled" "true" }}
```

Comment:
I believe it implies that we want Sensor to connect to any scanner (v2 or v4). I searched the codebase, and it looks like it is not tied to Scanner V2, but rather local/delegated scanning in general

Reply:
Okay, thanks. I was under the impression that this is only linked to v2.

Reply:
Nevertheless, since Scanner V4 enabled + Scanner 2 disabled is not supported, I would prefer to have the logic so we don't set local image scanning in this case. Currently, with Scanner V4 being opt-in and not having feature-parity with v2 I see it more like an add-on to the current stack. I would assume that having ROX_LOCAL_IMAGE_SCANNING=true for sensor without any Scanner V2 wouldn't do any good?

Reply:
I have added tests to describe exactly the behaviour I assume to be what we want.

Reply:
I agree. But we'll have to remember to set the `ROX_LOCAL_IMAGE_SCANNING` from the Scanner V4 templates, once Scanner V2 is gone.